lib/Utils.vala

```diff
@@ -332,5 +333,53 @@ namespace Gala
 
 			return texture;
 		}
+		/**
+		 * Returns the pixbuf that is used for resize buttons throughout gala at a
+		 * size of 36px
+		 *
+		 * @return the close button pixbuf or null if it failed to load
+		 */
+		public static Gdk.Pixbuf? get_resize_button_pixbuf ()
+		{
+			if (resize_pixbuf == null) {
+				try {
+					resize_pixbuf = new Gdk.Pixbuf.from_resource_at_scale (Config.RESOURCEPATH + "/buttons/resize.svg", -1, 36, true);
+				} catch (Error e) {
+					warning (e.message);
+					return null;
+				}
+			}
+
+			return resize_pixbuf;
+		}
+
+		/**
+		 * Creates a new reactive ClutterActor at 36px with the close pixbuf
+		 *
+		 * @return The resize button actor
+		 */
+		public static Clutter.Actor create_resize_button ()
+		{
+			var texture = new Clutter.Texture ();
```

Comment:
Replacing deprecated symbols is not part this. While there is already a wide usage of Clutter.Texture there is no harm in following the current style.